### DISCUSSION TITLE
Discourage localization of journal title

### DISCUSSION THREAD

**Discussion by fgnievinski** [pkp/pkp-lib#8207] [Proposals]
**Describe the problem you would like to solve**
Currently, OJS3 allows localizing (translating) the journal title. But this is discouraged by CrossRef: _"you need to pick one journal title and stick with it. And, the language of the journal title does not need to match the language of the article title."_
https://community.crossref.org/t/parallel-titles-for-a-given-issn/2183/4?u=fgnievinski

**Describe the solution you'd like**
Disable language-specific journal title by default; offer an option to enable it only by admins.

**Who is asking for this feature?**
Journal Editor

**Additional information**
Further discussion at:
https://github.com/pkp/pkp-docs/issues/871#issuecomment-953968295

**Comment by asmecher** [comment]
Tagging @AhemNason on this -- personally I suspect we're going to get conflicting advice on how to represent translated journal titles, with some of it varying by local convention (e.g. Canadian journals will often put both languages in a single field). CrossRef's perspective will be one of many.

**Comment by fgnievinski** [comment]
bilingual titles seem fine for CrossRef as long as they're entered in compound form, as in "Canadian Psychology/Psychologie canadienne". it should remain fixed and it can be in any language, not necessarily one of the content languages – it can be a latin phrase, a loanword, or another foreign name used for historical reasons. for example, it's not uncommon for a regional journal to be internationalized and start accepting submissions in english language only. my impression is the editorial decision to keep the journal title unchanged has to do more with the journal brand name awareness. all of this is to say if OJS wants to be strict about the journal title language, it'd have to allow users to define it independently of the content language. the present issue goes in the opposite direction and proposes to make journal titles language agnostic instead.
edit: related issue at https://github.com/pkp/pkp-docs/issues/871

**Comment by Vitaliy-1** [comment]
An example of the ISSN record of my University's journal: https://portal.issn.org/resource/ISSN/2519-8572. 
The journal's title in Cyrillic doesn't make much sense for English speaking readers, in order only to read it, it's transliterated into Latin. As Cyrillic and transliterated titles still don't make sense for non-Ukrainian readers, there is a parallel translated English title.

**Comment by fgnievinski** [comment]
I'm afraid the current scheme in OJS does more harm than good with respect to citations.  as the how-to-cite tool will show different journal titles for each language, some of the incoming or received citations may end up unaccounted for in journal-level metrics (impact factor, CiteScore, cites per doc, H-factor, etc.).  so, language specific journal titles are confusing not only for CrossRef but potentially also for all other academic indexes (Google Scholar, Scopus, Scimago, Web of Science, etc.).  maybe this is why commercial publishing platforms seem to stick with a fixed journal title.  even in case it's been transliterated or translated, the original title is no longer used in metadata.

**Comment by asmecher** [comment]
> maybe this is why commercial publishing platforms seem to stick with a fixed journal title

@fgnievinski, in my experience other platforms just haven't considered/accommodated multilingual content. That's always been a big priority for us. However, as you're seeing here, it's hard to come up with good policies that are representative of the global publishing community and play nice with standards and services.

@Vitaliy-1's example above is a good one -- where journals might publish in different character sets. Other languages with this issue (using potentially both Cyillic and Latin) are for example Uzbek and Belarusian. I suspect Japanese, Chinese, Korean, and other languages often want to be able to generate citations in Latin languages too.

**Comment by fgnievinski** [comment]
I wondered if journals in different character sets "eat their own dog food": do the references cited in their articles adopt their own language specific journal titles? for the journal instance at hand, even when the content is in Cyrillic script and Ukrainian language (Українська), the list of references use the journal title in Latin script and English language, "PMGP" – Psychosomatic Medicine and General Practice, instead of "Психосоматична медицина та загальна практика". 

so, a refinement of the present proposal would be for OJS to still allow language specific journal titles in the about page display but encourage adopting a fixed journal title in metadata. it corresponds indeed to what was implemented in this particular instance – both DC.Source and citation_journal_title are given in English ("Psychosomatic Medicine and General Practice") even though the rest of the metadata is in Ukrainian. citations are the currency of journal publishing, hence the proposal to discourage practices likely to cause trouble downstream. all the citation guides that I've consulted discourage changing the journal title:

- https://blog.apastyle.org/apastyle/translations/
> In the reference list, translate the title of the foreign-language work into the language you are writing in (here, that’s English). Otherwise, the details of the foreign-language source should stay as they were published, to aid in retrievability. Note for this example that Canadian Psychology/Psychologie canadienne is a bilingual journal that is published with a bilingual title; if the journal title were only in French it would not be necessary to translate it in the reference.
> > Bussières, E.-L., St-Germain, A., Dubé, M., & Richard, M.-C. (2017). Efficacité et efficience des programmes de transition à la vie adulte: Une revue systématique [Effectiveness and efficiency of adult transition programs: A systematic review]. _Canadian Psychology/Psychologie canadienne_, 58, 354–365. https://doi.org/10.1037/cap0000104 

- https://blog.apastyle.org/apastyle/2012/12/citing-translated-works-in-apa-style.html
> Here’s another example, from a German journal. Again, brackets contain an English translation of the work’s title (the article, not the journal).
> >  Janzen, G., & Hawlik, M. (2005). Orientierung im Raum: Befunde zu Entscheidungspunkten [Orientation in space: Findings about decision points]. _Zeitschrift für Psychologie_, 213(4),  179–186. doi:10.1026/0044-3409.213.4.179

example 1:
https://uk.e-medjournal.com/index.php/psp/article/view/326
cites:
Avramchuk O. Social anxiety disorder: relevance and perspectives. PMGP. 2018;3(3). DOI: https://doi.org/10.26766/pmgp.v3i3.103

example 2:
https://uk.e-medjournal.com/index.php/psp/article/view/324
cites:
Khaustova OO. Results of 2017: updated clinical guidelines. PMGP. 2017;2(4):e020496. URL: https://e-medjournal.com/index.php/psp/article/view/96

**Comment by asmecher** [comment]
See also this conversation:

https://academia.stackexchange.com/questions/1913/how-should-references-in-non-latin-script-language-be-treated-in-english-paper

The key distinction is between transliteration and translation, I think. Citations generally shouldn't be translated, but it can be very useful to have them transliterated (e.g. across Cyrillic and Latin character sets). Per the link above, many citation standards have provisions for this.

However, we use the `citeproc-php` library, which implements support based on Citation Style Language (CSL) templates for each citation format. While CSL does provide for `original-title` (the title of the article in its original form), what we're discussing is the `container-title` (the title of the journal). Unfortunately there is no `original-container-title` support in the CSL standard; that's been discussed here (https://discourse.citationstyles.org/t/expanded-short-original-reviewed-variables/1504) but not resolved.

**Comment by fgnievinski** [comment]
I think we can leave the problem of the exact citation format for CSL folks. The central issue here is, instead, metadata for improved discoverability. To recap, the present proposal is to offer an optional configuration letting the journal editor choose a master language for the journal title metadata. That way, HTML metadata tags and XML exports (and the single-language how-to-cite text) would use a fixed journal title; the About page could continue to use language specific versions. 

The journal title master language would be preferably either the translated or the transliterated version, but even the original non-Latin script version could be chosen, if the editor so prefers. In case the proposed configuration is left undefined, the standard usage remains unchanged: language specific journal titles are used in metadata. 

The end goal is to curb editors into following best practices from library science (in citations, journal titles should not be translated, only possibly transliterated) and from [CrossRef](https://www.crossref.org/documentation/content-registration/verify-your-registration/troubleshooting-submissions/#00634) ("if your journal is in more than one language, you need to choose one version of the title as the master entry").


PS: to better support transliterated metadata (in all of author names, article titles, and journal title), ideally OJS would support the optional four-letter script subtags in IETF language tags; this would allow the user to specify variations such as:
- uk-Cyrl: Ukranian language in original script
- uk-Latn: Ukranian language in Latin script (transliteration)
- en-Latn (or just en): English language in Latin script (translation)
- en-Latn-CA (or just en-CA): English language in Latin script, as used in Canada

Maybe OJS is agnostic about language subtags and already supports an arbitrary user input language tag? Background:
https://en.wikipedia.org/wiki/IETF_language_tag
https://en.wikipedia.org/wiki/ISO_15924

**Comment by NateWr** [comment]
Sorry to be a grump here. But in my view the problem is with the assumption that a single work is represented by a single citation string. That just doesn't reflect reality. This is a problem for indexers, not a problem for OJS.

**Comment by asmecher** [comment]
A note on locale codes for languages that offer several character set variants: we use the `xx_YY@charset` form, with `@charset` only included where necessary. You can see several examples here: https://github.com/pkp/pkp-lib/tree/main/locale

**Comment by fgnievinski** [comment]
> Sorry to be a grump here. But in my view the problem is with the assumption that a single work is represented by a single citation string. That just doesn't reflect reality. This is a problem for indexers, not a problem for OJS.

maybe not a single citation string, as it depends on the language and script of the citing document and the citation style, as per examples below.

but the journal title ought to be kept fixed to whatever the publisher chose to print on the front cover and decided to register with the ISSN authority, plus a transliteration if necessary.

so, a citation of the article at https://uk.e-medjournal.com/index.php/psp/article/view/326 in Chicago style would be:

1.) assuming the editor picked the master journal title as "Psychosomatic Medicine and General Practice":

1.1) for a cited article published in a single language and script (uk-Cyrl):

1.1.1) when the citing article is also written in uk-Cyrl: 

> Аврамчук, Олександр. “Клініко-психологічні особливості осіб з ознаками соціально тривожного розладу, який асоціюється з травматичним досвідом.” _Psychosomatic Medicine and General Practice_ 6, 4 (2021): e0604326.

1.1.2) when the citing article is written in uk-Latn: 

> Avramchuk, Oleksandr. Аврамчук, Олександр. Kliniko-psykholohichni osoblyvosti osoby z oznakamy sotsialʹnoho tryvozhnoho rozladu, yakyy asotsiyuyetʹsya z travmatychnym dosvidom. “Клініко-психологічні особливості осіб з ознаками соціально тривожного розладу, який асоціюється з травматичним досвідом.” _Psychosomatic Medicine and General Practice_ 6, 4 (2021): e0604326. 

1.1.3) when the citing article is written in en: 

> Avramchuk, Oleksandr Аврамчук, Олександр. Kliniko-psykholohichni osoblyvosti osoby z oznakamy sotsialʹnoho tryvozhnoho rozladu, yakyy asotsiyuyetʹsya z travmatychnym dosvidom. “Клініко-психологічні особливості осіб з ознаками соціально тривожного розладу, який асоціюється з травматичним досвідом.” [Clinical and psychological features of persons with social anxiety disorder associated with traumatic experience] _Psychosomatic Medicine and General Practice_ 6, 4 (2021): e0604326.

1.1.4) any possible multilingual metadata of a monolingual article is just a courtesy, it doesn't override the citation style rules; so, the English version should not be included alone in the citations, otherwise the reader might think the cited article was written in English:

> <strike>Avramchuk, Oleksandr. “Clinical and psychological features of persons with social anxiety disorder associated with traumatic experience”. _Psychosomatic Medicine and General Practice_ 6, 4 (2021): e0604326.</strike>

1.2) for a cited article published in multiple languages and/or scripts (not just multilingual metadata): pick the preferred version, just be careful with the pagination (which would not be the same for all versions).

2.) if the editor picked the master journal title in uk-Cyrl, "Психосоматична медицина та загальна практика", then "Psychosomatic Medicine and General Practice" would be replaced in the citations above for "Psykhosomatychna Medytsyna ta Zahalʹna Praktyka Психосоматична медицина та загальна практика", when the citing article is written in English or in uk-Cyrl.

**Comment by Vitaliy-1** [comment]
Regarding references in articles of this particular journal, it's just negotiations between editorial staff. 

If continue the discussion around this journal, I think you may notice that it publishes the same article in 2 languages, with 2 separate URLs. The articles in the journal are bilingual, e.g.: 

- English: https://e-medjournal.com/index.php/psp/article/view/92 
- Ukrainian: https://uk.e-medjournal.com/index.php/psp/article/view/92

If the article doesn't have a version published in one of the languages, there is a link, which points to the language version available. Handling citations as you mentioned would likely lead to more confusion.

**Comment by Vitaliy-1** [comment]
> it corresponds indeed to what was implemented in this particular instance – both DC.Source and citation_journal_title are given in English ("Psychosomatic Medicine and General Practice") even though the rest of the metadata is in Ukrainian

I think you identified a bug for this particular journal. While there are changes for serving multilingual content in separate URLs, Dublin Core Metadata Plugin wasn't taken into account. This OJS instance is quite heavily customized.

**Comment by Vitaliy-1** [comment]
> The central issue here is, instead, metadata for improved discoverability. To recap, the present proposal is to offer an optional configuration letting the journal editor choose a master language for the journal title metadata. That way, HTML metadata tags and XML exports (and the single-language how-to-cite text) would use a fixed journal title

To double-check this, the problem is only in `how-to-cite` text, right? or there are other places where in metadata tags or XML exports isn't used a fixed title?

**Comment by NateWr** [comment]
> or there are other places where in metadata tags or XML exports isn't used a fixed title?

Probably Crossref, which I think expects a journal title with each deposit. Not sure about others, but there's a list of distribution services in https://github.com/pkp/pkp-lib/issues/5980#issue-636172778.

**Comment by fgnievinski** [comment]
> > The central issue here is, instead, metadata for improved discoverability. To recap, the present proposal is to offer an optional configuration letting the journal editor choose a master language for the journal title metadata. That way, HTML metadata tags and XML exports (and the single-language how-to-cite text) would use a fixed journal title
> 
> To double-check this, the problem is only in `how-to-cite` text, right? or there are other places where in metadata tags or XML exports isn't used a fixed title?

For the sake of the argument, let's assume we agree it's a good idea to follow the ISSN and CrossRef data model and accept for the moment a journal cannot have more than one main title.

If the editor enters the journal title in multiple languages in OJS, in principle those language-specific versions will show up in all of HTML tags, XML exports, and how-to-cite text.  This would cause a more serious problem when the joural publishes multilingual articles.  In this case, the HTML tags (used by Google Scholar) and the XML exports (for CrossRef, DOAJ, etc.) will include language-specific versions of the journal title.  It'll raise a deposit error in CrossRef, Google Scholar may miss citations and miscalulate the journal H-factor, DOAJ may not realize the two titles are supposed to be the same one journal.  Language-specific journal titles in multilingual metadata of a monolingual article causes a less serious problem because crawlers ignore multilingual metadata, as OJS doesn't offer different URLs for exposing content versions in different languages. Only when a user manually selects a differernt language in the user interface they'd see the language-specific journal title versions.  The how-to-cite text is a minor problem, the major problem is the metadata in HTML tags and XML exports.

PS: the citations given as example follow Chicago style literally, any journal is free to adopt their own unique citation style, of course. But all classic citation styles (Chicago, APA, etc.) say the citation should reflect the original content language. A translation is a separate work, including different pagination and identifiers. If you cite both the original work and its translation in the same article the two citations are supposed to look different in the list of references (except for the journal title).

**Comment by AhemNason** [comment]
My perspective is that Crossref are reviewing multilingual metadata _as we speak_ and I think if we’re going to have large conversations about the treatment of multilingual metadata and its innumerable, potential downstream endpoints, we’ll have to do a much broader review than just what Crossref needs.

These issues go well beyond Crossref. Google scholar expects specific things for translated metadata, datacite has specific supports, orcid has specific supports, jats, Crossref... there doesn’t appear to be an international standard or recommendation for the treatment of these things. Crossref will be basing their solutions mostly off of JATS. And cataloguers in libraries have their own rules for how this work is recorded. And a lot of those rules are _very_ anglocentric or specifically intended to be unilingual. How are the stewards of open scholarly infrastructure solving these issues? 

I am increasingly uncomfortable with making executive decisions when most of the parties in this space are still writing their notes down in pencil. What I’d probably recommend is a significant review or study of multilingual metadata standards that would allow the many disparate groups trying to solve these problems to operate from the same space. 

We know that limiting options in this space for the sake of funneling users to doing _**the right thing**_ can often backfire and result in abuses of metadata to achieve the same means. Being restrictive about language metadata (even when it is, say, "for the journals own good") can have the outward effect of those users jamming multiple languages in one field). We know that no matter what we decide, we can't make a user do anything. We cannot _enforce_ metadata fidelity.

**Comment by fgnievinski** [comment]
Meanwhile, I've submitted a documentation patch describing Crossref's recommendations on journal titles to section "Adding multilingual metadata in OJS" of the "best practices for indexing" document:
https://github.com/pkp/pkp-docs/pull/937

> If you publish multilingual content and use DOIs from Crossref, also consider their [recommendation about journal titles](https://www.crossref.org/documentation/content-registration/verify-your-registration/troubleshooting-submissions/#00634): “if your journal is in more than one language, you need to choose one version of the title as the master entry”.  [In other words](https://community.crossref.org/t/parallel-titles-for-a-given-issn/2183/4): “you need to pick one journal title and stick with it. And, the language of the journal title does not need to match the language of the article title.” Otherwise, having a different journal title for each language may cause an error when submitting the DOIs for registration.
